### PR TITLE
possibility to configure location of key file

### PR DESCRIPTION
**Pull request by lgazo**

### CODE REVIEW

**Comment by jnovack** [comment]
Please rebase.

**Comment by Clement-TS** [comment]
@lgazo up?

**Comment by lgazo** [comment]
synced with upstream...

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,16 @@
 FROM alpine
-MAINTAINER Justin J. Novack <jnovack@gmail.com>
+MAINTAINER Ladislav Gazo <gazo@seges.sk>
```
Comment (ElijahLynn) [inline Dockerfile]: Need to revert this maintainer change if you want to get it merged.

**Comment by ElijahLynn** [comment]
I don't see the code change for this, only changes are readme instructions. Your latest merge overwrote your commits.

**Comment by jnovack** [comment]
Closing due to lack of interest by submitter.

**Pull request closed without merging**